t/op/for.t

```diff
@@ -548,12 +550,15 @@ for my $i (reverse (map {$_} @array, 1)) {
 }
 is ($r, '1CBA', 'Reverse for array and value via map with var');
 
-is do {17; foreach (1, 2) { 1; } }, '', "RT #1085: what should be output of perl -we 'print do { foreach (1, 2) { 1; } }'";
+{
+    no warnings 'void';
+    is do {17; foreach (1, 2) { 1; } }, '', "RT #1085: what should be output of perl -we 'print do { foreach (1, 2) { 1; } }'";
```

Comment:
Can you re-break this line at the description so that it's not running > 120 characters long?

Otherwise, this is great.

Reply:
Yes, good point. Done and now in blead.